src/LibraryUtilities.ts

```diff
@@ -566,48 +285,244 @@ export function updateSections(oldLayoutSpecs: any, newLayoutSpecs: any): void {
 }
 
 /**
- * Convert an array of typeListNodes to ItemData which are not processed based on their fullyQualifiedNames, 
- * by splitting the name with '.', and put them under a specific section.
+ * This function will convert layoutElements to ItemData, and construct an array of IncludeItemPair,
+ * which stores an includeInfo and its parent item.
  * 
- * As an example, for two typeListNodes 'A.B.C' and 'A.B.D', the resulting nodes are:
- * - A
- *   |- B
+ * @param {LayoutElement[]} layoutElements 
+ * The layoutElements to be converted to ItemData
```

Comment:
Since we are referring to the objects by its type, then a capital `LayoutElement` should be used:

`The list of LayoutElement objects to be converted to ItemData`